server/app_prod.yaml

```diff
@@ -12,11 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: python37
+runtime: python
+env: flex
+entrypoint: gunicorn -b :$PORT main:app
 
-instance_class: F4_1G
-automatic_scaling:
-  min_instances: 3
+runtime_config:
+  python_version: 3
+
+manual_scaling:
+  instances: 10
```

Comment:
should we get more instances for prod vs staging?

Reply:
Yeah. I don't know a good number to set for now. Might let it run for a while and adjust if we saw performance issues?

Reply:
ok